sequoia/settings/offline_rl/setting.py

```diff
@@ -0,0 +1,76 @@
+import d3rlpy
+from sklearn.model_selection import train_test_split
+from typing import ClassVar, List, Tuple, Dict
```

Comment:
```suggestion
from typing import ClassVar, List, Tuple, Dict, Optional
```

Reply:
I just removed the metrics completely and made it a class var of the method, that way we can log them or do what we want, but they aren't assumed by OfflineRLSetting

Reply:
Also moved testing to setting. I kept the test function in BaseOfflineRLMethod bc TraditionalRLSetting calls it for some reason (it breaks if I don't wrap the input env and run the test myself).